corehq/motech/fhir/models.py

```diff
@@ -298,3 +303,29 @@ def get_resource_type_or_none(case, fhir_version) -> Optional[FHIRResourceType]:
         )
     except FHIRResourceType.DoesNotExist:
         return None
+
+
+class FHIRImporter(models.Model):
+    domain = models.CharField(max_length=127, db_index=True)
+    connection_settings = models.ForeignKey(
+        ConnectionSettings,
+        on_delete=models.PROTECT,
+    )
+    fhir_version = models.CharField(
+        max_length=12,
+        choices=FHIR_VERSIONS,
+        default=FHIR_VERSION_4_0_1,
+    )
+    frequency = models.CharField(
+        max_length=12,
+        choices=IMPORT_FREQUENCY_CHOICES,
+        default=IMPORT_FREQUENCY_DAILY,
+    )
+    # ID of user or location that will own imported cases
+    owner_id = models.CharField(max_length=32, null=False, blank=False)
+
+    class Meta:
+        indexes = [
+            models.Index(fields=['domain']),
+            models.Index(fields=['frequency']),
+        ]
```

Comment:
What's the difference between specifying indices here versus on the model fields?  I see `domain` already sets `db_index=True`, but `frequency` doesn't - what's that about?

Reply:
Thank you for spotting! This will actually result in two indices for "domain"!

I think one would use `Meta.indexes` for more control over index options, and/or to be clearer about which fields are indexed. The index options aren't useful here, but I quite like being clear, because the index on `frequency` might not be obvious.